### DISCUSSION TITLE
How to clear the cache for the Keyv?

### DISCUSSION THREAD

**Discussion by BarryAllen-Arrow** [waylaidwanderer/node-chatgpt-api#284] [Help]
Hi there! I'm using jail breaked bing, so there's a keyv method for it, although it's always working, it's quite confusing. Sometimes, the cache would save down in the cache.json file as I set. But when I copy another instance of the "use-bing-client.js" and give the keyv file a different name, like cache2.json, it wouldn't save down anything! So I was thinking if I just clear everything in the cache.json file, I can clear the cache, right? But since it's not storing stuffs right now, I have no idea how to clear cache.

And there's another stupid question, will the cache affect the speed or perform of bing? if there're a large amount of quesions stored in the cache, will it get bing respond slower?

**Comment by waylaidwanderer** [comment]
You can manually clear it by just deleting everything inside cache.json. The file itself still needs to exist.

The keyv file store is probably slower compared to using something like redis or MySQL, but it shouldn't be a big issue.
